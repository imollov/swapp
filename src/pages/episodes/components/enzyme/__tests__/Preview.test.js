import React from 'react'
import { shallow } from 'enzyme'

import EpisodePreview from '../../Preview'

describe('EpisodePreview Component', () => {
  it('smoke test', () => {
    const wrapper = shallow(
      <EpisodePreview img="image.png" title="title" text="text" />,
    )

    expect(wrapper).toMatchSnapshot()
  })
})
