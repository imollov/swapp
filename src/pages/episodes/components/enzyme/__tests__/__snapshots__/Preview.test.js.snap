// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EpisodePreview Component smoke test 1`] = `
<Styled(styled.div)
  flexDirection="column"
  variant="card"
>
  <BackgroundImage
    height={256}
    src="image.png"
  />
  <styled.div
    p={3}
  >
    <ForwardRef
      py={3}
      variant="h2"
    >
      title
    </ForwardRef>
    <ForwardRef
      variant="body"
    >
      text
    </ForwardRef>
  </styled.div>
</Styled(styled.div)>
`;
